src/generate_sudoku_template.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+
+/*
+*File name: generate_sudoku_templete.c
+*
+*Description: This function generates a sudoku templete. 
+*
+*
+*Author: Rishith Rao
+*
+*
+**/
+
+
+void generate_sudoku_templete(int sudoku_temp[][9]){
```

Comment:
The function name is misspelled.
The name should be : general_sudoku_templ**a**te(...)